README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion : recurrence or repetition,
+- Recursive Case: repeated application of a recursive statement
```

Comment:
This is slightly strange wording to me. 

A recursive case is the case in which the method calls itself.